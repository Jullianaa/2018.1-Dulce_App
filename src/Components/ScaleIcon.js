import React from 'react';
import {Image, TouchableHighlight} from 'react-native';

const styles = {
  container: {
    marginTop: 100,
    width: 100,
    justifyContent: 'center',
    alignItems: 'center'
  },
  icon: {
    height: 100,
    width: 100
  }
};

class ScaleIcon extends React.Component {

  render() {
    return (
      <TouchableHighlight onPress = {() => this.props.onPress()} style = {styles.container}>
        <Image source={require('../../assets/img/calendarIcon.png')} style = {styles.icon}/>
      </TouchableHighlight>
    );
  }
}

export default ScaleIcon;
