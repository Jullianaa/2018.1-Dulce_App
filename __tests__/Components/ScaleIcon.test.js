import React from 'react';
import ScaleIcon from '../../src/Components/ScaleIcon';
import Adapter from 'enzyme-adapter-react-16';
import {shallow} from 'enzyme';
import Enzyme from 'enzyme';
import renderer from 'react-test-renderer';
require('bezier');

Enzyme.configure({adapter: new Adapter()});

it('renders correctly', () => {
  const tree = renderer
  .create(<ScaleIcon />)
  .toJSON();
  expect(tree).toMatchSnapshot();
});
