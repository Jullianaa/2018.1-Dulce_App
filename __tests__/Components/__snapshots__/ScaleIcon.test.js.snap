// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  hasTVPreferredFocus={undefined}
  hitSlop={undefined}
  isTVSelectable={true}
  nativeID={undefined}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "justifyContent": "center",
      "marginTop": 100,
      "width": 100,
    }
  }
  testID={undefined}
  tvParallaxProperties={undefined}
>
  <Image
    source={
      Object {
        "testUri": "../../../assets/img/calendarIcon.png",
      }
    }
    style={
      Object {
        "height": 100,
        "width": 100,
      }
    }
  />
</View>
`;
